### PR TITLE
Create concept entry for HEX function for MySQL

### LINKED ISSUE #5023: [Concept Entry] MySQL HEX function HEX()
### Reason/inspiration (optional)

We would like a new entry on the `HEX` concept under MySQL. The entry should go in a new file under `docs/content/mysql/concepts/hex/hex.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept (if applicable)
- An `Example` section that provides an example demonstrating the concept in use (if applicable)
  
Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by ebikatsudon**
### Description

Created documentation for the MySQL hex function. Describes the function's usage, syntax, and includes some examples in action.

### Issue Solved

Closes #5023 

### Type of Change

- Adding a new entry

### Checklist

- [ x ] All writings are my own.
- [ x ] My entry follows the Codecademy Docs style guide.
- [ x ] My changes generate no new warnings.
- [ x ] I have performed a self-review of my own writing and code.
- [ x ] I have checked my entry and corrected any misspellings.
- [ x ] I have made corresponding changes to the documentation if needed.
- [ x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by cigar-galaxy82** [comment]
Hi @ebikatsudon can you please sign the CLA

**Comment by ebikatsudon** [comment]
Sorry about that -- I had forgotten to add the email used for the commit to my Github account. I have signed the CLA; please let me know if there is anything else I need to do. Thank you!

File: content/mysql/concepts/hex/hex.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  - 'Strings'
+  - 'Database'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The **HEX()** function in MySQL takes an input and returns its equivalent hexadecimal representation. This input can be either a string or numeric value.
+
+If the input is a string, the resulting hexadecimal representation will, by default, be a binary string. Each byte of each character in the original string argument is represented by a corresponding pair of hexadecimal digits in the output. The resulting hexadecimal string can also be converted back into the original string by using the `UNHEX()` function.
```
Comment (cigar-galaxy82) [inline content/mysql/concepts/hex/hex.md:20]:
```suggestion
When the input is a string, the resulting hexadecimal representation will, by default, be a binary string. In the output, each byte of each character in the original string argument is represented by a corresponding pair of hexadecimal digits. The resulting hexadecimal string can also be converted back into the original string by using the `UNHEX()` function.
```

File: content/mysql/concepts/hex/hex.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+On the other hand, if the input is a decimal number, the resulting hexadecimal will be treated as a longlong (BIGINT) number.  In this scenario where N is a decimal number, the function `HEX(N)` produces the same output as the `CONVERT()` function, `CONV(N, 10, 16)`. This means that the hexadecimal output of `HEX(N)` can also be converted back to its original number by running `CONV(HEX(N), 16, 10)`.
```
Comment (cigar-galaxy82) [inline content/mysql/concepts/hex/hex.md:22]:
```suggestion
When the input is a decimal number, HEX(N) and CONV(N, 10, 16) produce the same hexadecimal output. The hexadecimal result from HEX(N) can be converted back to the original number using CONV(HEX(N), 16, 10).
```

File: content/mysql/concepts/hex/hex.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  - 'MySQL'
+  - 'Functions'
+  - 'Strings'
+  - 'Database'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The **HEX()** function in MySQL takes an input and returns its equivalent hexadecimal representation. This input can be either a string or numeric value.
```
Comment (cigar-galaxy82) [inline content/mysql/concepts/hex/hex.md:18]:
```suggestion
The **`HEX()`** function in MySQL takes an input and returns its equivalent hexadecimal representation. This input can be either a string or a numeric value.
```

File: content/mysql/concepts/hex/hex.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+('Brown', 64695),
+('Miller', 23088),
+('Williams', 15312);
+```
+
+To verify the data was inserted correctly, run the following.
+
+```sql
+SELECT * FROM employees;
+```
```
Comment (cigar-galaxy82) [inline content/mysql/concepts/hex/hex.md:85]: Put this in a single code block, consisting of creating the table, inserting and verifying the data

File: content/mysql/concepts/hex/hex.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+The resulting table looks like:
```
Comment (cigar-galaxy82) [inline content/mysql/concepts/hex/hex.md:87]:
```suggestion
The resulting table looks like this:
```

File: content/mysql/concepts/hex/hex.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
+---
+Title: 'Hex()'
```
Comment (cigar-galaxy82) [inline content/mysql/concepts/hex/hex.md:2]:
```suggestion
Title: 'HEX()'
```

File: content/mysql/concepts/hex/hex.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+Description: 'Returns a hexidecimal string representation of a given string or numeric argument.' 
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+  - 'Data Science'
+  Tags:
+  - 'MySQL'
+  - 'Functions'
+  - 'Strings'
+  - 'Database'
```
Comment (cigar-galaxy82) [inline content/mysql/concepts/hex/hex.md:12]:
```suggestion
  - 'Functions'
  - 'Database'
  - 'MySQL'
  - 'Strings'
  - 'Numbers'
```

File: content/mysql/concepts/hex/hex.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
+---
+Title: 'Hex()'
+Description: 'Returns a hexidecimal string representation of a given string or numeric argument.' 
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+  - 'Data Science'
+  Tags:
```
Comment (cigar-galaxy82) [inline content/mysql/concepts/hex/hex.md:8]:
```suggestion
Tags:
```

File: content/mysql/concepts/hex/hex.md [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+  - 'paths/analyze-data-with-sql'
+---
+
+The **HEX()** function in MySQL takes an input and returns its equivalent hexadecimal representation. This input can be either a string or numeric value.
+
+If the input is a string, the resulting hexadecimal representation will, by default, be a binary string. Each byte of each character in the original string argument is represented by a corresponding pair of hexadecimal digits in the output. The resulting hexadecimal string can also be converted back into the original string by using the `UNHEX()` function.
+
+On the other hand, if the input is a decimal number, the resulting hexadecimal will be treated as a longlong (BIGINT) number.  In this scenario where N is a decimal number, the function `HEX(N)` produces the same output as the `CONVERT()` function, `CONV(N, 10, 16)`. This means that the hexadecimal output of `HEX(N)` can also be converted back to its original number by running `CONV(HEX(N), 16, 10)`.
+
+ ## Syntax
```
Comment (cigar-galaxy82) [inline content/mysql/concepts/hex/hex.md:24]:
```suggestion
## Syntax
```

File: content/mysql/concepts/hex/hex.md [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+
+The expected output would then be:
+
+| last_name | HEX(last name)   |
+| --------- | ---------------- |
+| Smith     | 536D697468       |
+| Jones     | 4A6F6E6573       |
+| Brown     | 42726F776E       |
+| Miller    | 4D696C6C6572     |
+| Williams  | 57696C6C69616D73 |
```
Comment (cigar-galaxy82) [inline content/mysql/concepts/hex/hex.md:113]: Give one line after this

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
Hi @ebikatsudon thank you for contributing to Codecademy docs, I have suggested a few suggestions please modify

**Comment by ebikatsudon** [comment]
Hello @cigar-galaxy82, I have made the requested changes. Please let me know if you have any additional comments or suggestions. Thank you!

**Comment by ebikatsudon** [comment]
Hello, just wanted to double-check if there's anything else I need to do for this? @cigar-galaxy82

**Comment by mamtawardhani** [comment]
@cigar-galaxy82 could you please check this on priority? It's been two weeks to this. Thank you

**Review by cigar-galaxy82 — APPROVED** [review]

**Review by Sriparno08 — APPROVED** [review]
Looks good, @ebikatsudon!

**Pull request merged**